.github/workflows/build-and-test-all.yml

```diff
@@ -34,23 +44,21 @@ jobs:
         run: |
           echo "stamp=$(/bin/date +%s)" >> "$GITHUB_OUTPUT"
         shell: bash
+      - run: mkdir -p ~/.ccache
       - name: let GitHub cache our ccache data
         uses: actions/cache@v3
         with:
           path: ~/.ccache
           key: auth-ccache-${{ steps.get-stamp.outputs.stamp }}
           restore-keys: auth-ccache-
-      - run: build-scripts/gh-actions-setup-inv  # this runs apt update+upgrade
-      - run: inv install-clang
       - run: inv install-clang-tidy-tools
-      - run: inv install-auth-build-deps
```

Comment:
Where did those go? I'm guessing into the base image? If yes, can we also just install the clang-tidy-tools in the base image instead of installing them for each product?

Reply:
Yes, the idea is that all those packages were already installed while building the base image. I will add that step also there and remove it from here